docs/encoding.md

```diff
@@ -93,6 +94,30 @@ In the example above, we run an XSD validator before parsing the XML into an arr
 
 More information about [the PHP format can be found here](#php-format).
 
+#### element_encode
+
+Encode transforms an array into an XML string.
```

Comment:
Same here: looks like a better idea to differentiate and link between the two functions at the top.